deepchecks/tabular/_shared_docs.py

```diff
@@ -33,18 +33,18 @@
 """.strip('\n')
 
 _shared_docstrings['feature_aggregation_method_argument'] = """
-argument for the reduce_output functionality, decides how to aggregate the drift scores for a
+argument for the reduce_output functionality, decides how to aggregate the per-feature scores into a
```

Comment:
Maybe add italics to the "per-feature scores" and remind that they are a vector of a score per-feature?